jib-core/src/test/java/com/google/cloud/tools/jib/event/EventHandlersTest.java

```diff
@@ -53,46 +45,47 @@ private void sayHello(String name) {
     }
   }
 
-  /** Test class to hold mutable int. */
-  private static class Counter {
-
-    private int count = 0;
-  }
-
   @Test
   public void testAdd() {
-    Counter counter = new Counter();
+    int[] counter = new int[1];
```

Comment:
Assuming you're just using this so you can pass by reference, maybe an `Integer` would look less weird?

Reply:
I think `Integer` is immutable, once set. Here we want to increment values, so I think it won't work that way.

Reply:
Yea, I think the array method should be fine here since this is just a test and this array method is a common way to get around the effectively final requirement for lambdas.

Reply:
Ah ok, didn't know Integer was immutable.